### DISCUSSION TITLE
Daily: Run on weekend too

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#15901] [Code Security]
Why does dependabot only run on weekdays? A bot could run everyday, including weekends.

From the docs:
```
daily—runs on every weekday, Monday to Friday.
```
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval

**Comment by rwojsznis** [comment]
Was surprised this is not configurable as well 🤔 you shouldn't really [assume](https://en.wikipedia.org/wiki/Workweek_and_weekend) that workweek always happens between Monday and Friday in the first place 🙈

**Reply by per1234** [reply]
There is a proposal here that would add the needed flexible configuration: https://github.com/dependabot/dependabot-core/issues/6339

**Comment by captn3m0** [comment]
I'd raised this with the GitHub Team at a community meetup, and they were surprised (hadn't heard this feedback before).

Another usecase is that "open-source" does not necessarily happen on a workweek - open-source projects need daily builds, as that work often happens outside of the regular workweek.

**Reply by isaacbowen** [reply]
Oh fantastic! I'm so glad you got to share this with them! Thank you for doing that!

_One thousand percent_ agreed with the point about daily builds being normal for open-source. (Also, many-other-kinds-of-source.)

**Comment by captn3m0** [comment]
Update: This should be resolved now with cron expressions getting supported: https://github.blog/changelog/2025-04-22-dependabot-now-lets-you-schedule-update-frequencies-with-cron-expressions/

It doesn't seem to be documented yet, so I've filed an issue: https://github.com/github/docs/issues/37712
